content/en/data_jobs/airflow.md

```diff
@@ -236,7 +236,64 @@ Check that the OpenLineage environment variables are correctly set on the Astron
 
 **Note**: Using the `.env` file to add the environment variables does not work because the variables are only applied to the local Airflow environment.
 {{% /tab %}}
+{{% tab "Google Cloud Composer" %}}
+<div class="alert alert-warning">
+Data Jobs Monitoring for Airflow is not yet compatible with <a href=https://cloud.google.com/composer/docs/composer-2/lineage-integration>Dataplex</a> data lineage. Setting up OpenLineage for Data Jobs Monitoring overrides your existing Dataplex transport configuration.
+</div>
+
+## Requirements
+
+* [Cloud Composer 2][1] or later
+* [apache-airflow-providers-openlineage][2]
+
+## Setup
+
+To get started, follow the instructions below.
 
+
+1. In the Advanced Configuration tab, under "Airflow configuration override", click on "Add Airflow configuration override" and configure as follows:
+
+   - In Section 1, enter `openlineage`
+   - In Key 1, enter `transport`
+   - In Value 1, enter the follows:
+     ```text
+     {
+      "type": "http", 
+      "url": "<DD_DATA_OBSERVABILITY_INTAKE>", 
+      "auth": {
+         "type": "api_key", 
+         "api_key": "<DD_API_KEY>"
+      }
+     }
```

Comment:
Looks like this code block is missing the ending syntax. Also, it'll render a little better with a blank line between the previous line and the start of the block:
```suggestion

     ```text
     {
      "type": "http", 
      "url": "<DD_DATA_OBSERVABILITY_INTAKE>", 
      "auth": {
         "type": "api_key", 
         "api_key": "<DD_API_KEY>"
      }
     }
     ```
```